samples/SocketsSample/Hubs/Chat.cs

```diff
@@ -3,7 +3,6 @@
 
 using System;
 using System.Threading.Tasks;
-using System.Collections.Generic;
```

Comment:
What's the deal here?

Reply:
Someone added it spuriously or forgot to organize and sort earlier.